user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -0,0 +1,137 @@
+---
+uid: Register_Catalog_Item
+---
+
+# Register your catalog item
+
+## What is a catalog item?
+
+The content available in the catalog module has always been provided by Skyline Communications. We have been using this module initially to share our connectors with all of our users. In a later stage we have been adding App Packages, Automation Scripts and Visual Overviews. All of these items are called "catalog items".
+
+As we see more and more users create their own customized solutions on top of the DataMiner software, we want to allow them to register their solutions to the catalog as well. This will enable our users to easily manage the solutions and packages they have created, distribute them within their organization between different teams and even share them with different organizations.
+
+## Private catalog items
+
+All of the items that have been provided by Skyline Communications are considered to be public items. They are available for all our users. Note that for some of these items (eg Connectors) the user might need to buy a license to be able to use them on their system.
+
+Private catalog items are items that are only available to one or more organizations. This gives users the possibility to develop App Packages, Automation Scripts or any type of catalog item and redeploy them within their organization on various systems or share them with other organizations.
+
+## Versioning of catalog items
+
+Catalog items can hold multiple versions. To make sure that the versioning of items is easy to understand for everyone [Semver](https://semver.org/) versioning is promoted for catalog item versions. Users can assign extra labels to versions if they want to indicate a certain version is not an official release (eg 1.2.3-alpha).
+
+## How to register a private catalog item
+
+### Register using the Catalog UI
+
+If you use the UI, you can only register a catalog item. There is no UI yet to register a new version, this can only be done by using the API.
+
+Registering a new catalog item can be done by executing the following steps.
+
+1. Go to the [Catalog](https://catalog.dataminer.services). There you will find a button next to the organization selector called         "Register Catalog". Clicking this button will open a popup with a few fields.
```

Comment:
looks like some spaces need to be cleaned away here.
"Selector called            "Register Catalog".